libraries/chain/include/golos/chain/steem_evaluator.hpp

```diff
@@ -81,4 +81,18 @@ namespace golos { namespace chain {
             int depth_ = 0;
         };
 
+        class recover_account_evaluator : public evaluator_impl<recover_account_evaluator> {
+        public:
+            using operation_type = recover_account_operation;
+
+            recover_account_evaluator(database& db) : evaluator_impl<recover_account_evaluator>(db) {}
+
+            void do_apply(const operation_type& o);
+
+        private:
+            void reset_vesting_withdraw(const account_object& account);
```

Comment:
Do not split code to many methods. Write whole code in do_apply.
It also allows we use standard DEFINE_EVALUATOR.

Reply:
It's bad practice to create huge methods filled by low-level logic, so that it is difficult to understand what this method is doing. Using decomposition and splitting method on sub-tasks (then each subtask also can be splitted) makes code more understandable: you see the top-level logic of this method and if it necessary to understand some operation logic - opens it methods an so on.

Reply:
@e-schepachev Let it OK. But not-using standard features is bad practice, too. In this case, standard way is DEFINE_EVALUATOR. We should prefer it always. About functions, again:
> If you really need separate function in evaluator - write in in cpp-file, as a function, not as a class method.

It is compatible with DEFINE_EVALUATOR.

Reply:
Ok. I like that we can see all evaluators more than dislike define_evaluator implementation.